pkg/common/cns-lib/volume/manager.go

```diff
@@ -219,8 +218,8 @@ type createVolumeTaskDetails struct {
 // GetManager returns the Manager instance.
 func GetManager(ctx context.Context, vc *cnsvsphere.VirtualCenter,
 	operationStore cnsvolumeoperationrequest.VolumeOperationRequest,
-	idempotencyHandlingEnabled bool, multivCenterEnabled bool,
-	multivCenterTopologyDeployment bool, tasksListViewEnabled bool) (Manager, error) {
+	idempotencyHandlingEnabled, multivCenterEnabled, multivCenterTopologyDeployment, tasksListViewEnabled bool,
```

Comment:
This PR adds `clusterFlavor` field to the manager struct so that the APIs can access it easily.